tests/data/program_analysis/select_case/select03.f

```diff
@@ -0,0 +1,25 @@
+C FORTRAN test file to test CASE constructs
+C This file tests an integer with a list of 'or' test cases in a single case
+
+      PROGRAM MAIN
+
+      IMPLICIT NONE
+
+      INTEGER :: I = 5
+      INTEGER :: X = 40
+      INTEGER :: Y
+      INTEGER :: Z = 2
+
+      SELECT CASE(I)
```

Comment:
Would it be possible to put a loop around the select-case so that all of the various possibilities in the CASE labels get exercised?  E.g., for `CASE (:2, 5, 9:)` I'd like to have (i) a value that exercises `:2`, (ii) a value  that exercises `5`, and (iii) a value that exercises `9:`; and similarly for the other case labels.

Reply:
I'll make the changes.

Reply:
Modified the test file.